### DISCUSSION TITLE
anything for how to detect audit log update?

### DISCUSSION THREAD

**Discussion by SSS-Says-Snek** [Rapptz/discord.py#6049] [Q&A] [answered]
Is there any way of detecting when an audit log updated? I want to make a bot that detects when the audit log updates, and send them to me. If there isn't any way, should there be an on_audit_update() event?

**Comment by Rapptz** [comment, marked as answer]
Discord does not provide gateway events that mention when an audit log entry has been added. There is the [`on_member_ban`](https://discordpy.readthedocs.io/en/latest/api.html#discord.on_member_ban) and [`on_member_unban`](https://discordpy.readthedocs.io/en/latest/api.html#discord.on_member_unban) but these don't have to coincide with the entry being published in the audit log (i.e. if you poll when you get these events they might not be in the audit log yet).

**Reply by SSS-Says-Snek** [reply]
OK, thanks!
